src/lib.rs

```diff
@@ -92,12 +118,18 @@
 //!
 //! // Loaded example image is 8x8px
 //! assert_eq!(pixels.len(), 8 * 8);
+//!
+//! // Individual raw pixel values can also be read
+//! let pixel = bmp.pixel(Point::new(3, 2));
+//!
+//! // The raw value for a white pixel in the source image
+//! assert_eq!(pixel, Some(16777215u32));
```

Comment:
Because the image is a 24 bpp image the color could be expressed as a hex color:
```suggestion
//! assert_eq!(pixel, Some(0xFFFFFFu32));
```